sbin/firehol

```diff
@@ -4358,10 +4358,10 @@ mac() {
 	if running_ipv6; then
 		if [ ${wrongmac6_chain} -eq 0 ]
 		then
-			set_work_function "Creating the MAC-MISSMATCH chain (only once)"
+			set_work_function "Creating the MAC-MISMATCH chain (only once)"
 
 			ip6tables -t filter -N WRONGMAC
-			rule table filter chain WRONGMAC loglimit "BLOCKED MAC MISSMATCH" action DROP || return 1
+			rule table filter chain WRONGMAC loglimit "BLOCKED MAC MISMATCH" action DROP || return 1
```

Comment:
We can't really change this in case someone is depending on the current behaviour.

Reply:
Reverted.